charms/worker/k8s/templates/snap_installation.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+amd64:
+- name: k8s
+  install-type: store
+  channel: edge
+arm64:
+- name: k8s
+  install-type: store
+  channel: edge
```

Comment:
by default, the `main` branch will continue installing from the edge channel.  Other branches will use specific snap revisions